app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/ProfileScreen.kt

```diff
@@ -68,11 +67,10 @@ val INPUT_BOX_NAM_SIZE = 150.dp
  * @last update 1.0
  */
 @Composable
-fun ProfileScreen(viewModel: ProfileViewModel = hiltViewModel()) {
+fun ProfileScreen(viewModel: ProfileViewModel) {
```

Comment:
Here, I meant refactor
``
@Composable
fun ProfileScreen() {
  val viewModel: ProfileViewModel = hiltViewModel()
  /**/
}
``
like so.

Reply:
Oh I see now, but the thing is that maybe later, we will connect few things that are based on the profile, like the authentication screen, ... , and I thought it was better to give it as an argument.
Because if we are using the profileViewmodel in other classes, can we still share the data without passing it as an argument?

Reply:
I think we are able to do so. we can just do 
`val viewModel: ProfileViewModel = hiltViewModel()` in the places that need it
and hilt will take care of the rest.